dds/src/dds/configuration.rs

```diff
@@ -8,6 +11,7 @@ pub struct DustDdsConfiguration {
     interface_name: Option<String>,
     fragment_size: usize,
     udp_receive_buffer_size: Option<usize>,
+    participant_announcement_interval: Time,

```

Comment:
could be also std::time::Duration

Reply:
Time was indeed a bad choice. Could be an std Duration or a dust_dds Duration.

Reply:
Since this is for configuration purposes we decided for an std::time::Duration